src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,13 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React from 'react';
+import Todo from './Todo';
+
+function TodoList(props) {
+    return (
+        <div className="list">
+            <h1>To Do:</h1>
+            {props.todoArray.map(x => <Todo completedToggle={props.completedToggle} task={x.task} key={x.id} id={x.id} />)}
```

Comment:
This would do better split up for readability as well:
```javascript
{props.todoArray.map(x => <Todo 
  completedToggle={props.completedToggle} 
  task={x.task} 
  key={x.id} 
  id={x.id} 
/>)}
```